internal/pkg/pct/pct.go

```diff
@@ -154,6 +161,17 @@ func FormatTemplates(tmpls []PuppetContentTemplate, jsonOutput string) error {
 	return nil
 }
 
+// DisplayDefaults returns the Default values from a Templates configuration file
+func DisplayDefaults(defaults map[string]interface{}) string {
+	prettyDefaults, err := yaml.Marshal(defaults)
+	if err != nil {
+		log.Error().Msgf("Error converting to json: %v", err)
+
+	}
+
+	return string(prettyDefaults)
```

Comment:
I think it would be awesome if it returned both template information as well as the config data. To me, the `info` about a template includes the name and other details.

```
❯ go run main.go new --info puppet-module --templatepath C:\Users\james\pct-examples\
DisplayName:     Puppet Module
Name:                 puppet-module
TemplateType:    project
TemplateURL:     https://github.com/puppetlabs/pct-puppet-module
TemplateVersion: 0.1.0

puppet_module:
  license: Apache-2.0
  operatingsystem_support:
  - operatingsystem: windows
    release:
    - 2019
    - 10
  - operatingsystem: ubuntu
    release:
    - "20.04"
    - "18.04"
  - operatingsystem: CentOS
    release:
    - "7"
  - operatingsystem: OracleLinux
    release:
    - "7"
```

Reply:
After discussion with @da-ar this doesn't make sense at this point. It would require modifying too much of the existing code right before a release. We'll revisit this after LA